doc/connecting/topics/as_connecting-to-google-sheets.adoc

```diff
@@ -0,0 +1,54 @@
+// This assembly is included in the following assemblies:
+// Upstream: connecting/master.adoc
+// Downstream: connecting-fuse-online-to-applications-and-services/master.adoc
+
+:context: connectors
+[id='connecting-to-google-sheets_{context}']
+= Connect to Google Sheets
+:context: sheets
+
+You can add a Google Sheets connection to an integration as its start
+connection to trigger execution of an integration. These integration start connections
+are periodic queries of spreadsheet data. You can add a Google
+Sheets connection to an integration as its finish connection when you want
+to update spreadsheet properties, update data in a spreadsheet, create charts, create pivot tables. In the middle of an integration, you can add a
+Google Sheets connection when you want to create a new spreadsheet, obtain particular spreadsheet properties,
+add or update data in a spreadsheet, create charts or create pivot tables.
+
+Details for connecting to Google Sheets are in the following topics:
+
+* <<register-with-google-sheets_{context}>>
+* <<enable-google-sheets-api_{context}>>
+* <<create-google-sheets-connection_{context}>>
+* <<add-google-sheets-connection-create-spreadsheet_{context}>>
+* <<add-google-sheets-connection-get-spreadsheet_{context}>>
+* <<add-google-sheets-connection-get-spreadsheet_{context}>>
```

Comment:
Should that be
<<add-google-sheets-connection-update-spreadsheet_{context}>>  ???

Reply:
totally! I will fix this